playground/static/main.js

```diff
@@ -265,22 +271,43 @@ async function updateOutput(input) {
                 errors.forEach(addError);
                 warnings.forEach(addWarning);
 
+                overleafButton.style.display = "none";
+
                 debugEditor.container.style.display = "block";
                 renderIframe.style.display = "none";
                 render.style.display = "none";
 
```

Comment:
Maybe remove this newline to improve style, so that all "set style"-statements stick together